appserver/ejb/ejb-internal-api/src/main/java/org/glassfish/ejb/api/EJBInvocation.java

```diff
@@ -47,83 +47,99 @@
 
 /**
  * This interface provides access to the exported portions of the
- * ejb invocation object.  
+ * ejb invocation object.
  * @author Kenneth Saks
  */
-
-
 public interface EJBInvocation {
 
-    public EJBContext getEJBContext();
-    
+    /**
+     * @return runtime {@link EJBContext} of this invocation
+     */
+    EJBContext getEJBContext();
```

Comment:
In Java all methods in interfaces are public by default and nobody can change it.
I added also some javadocs.